kubernetes/cray-csm-barebones-recipe-install/Chart.yaml

```diff
@@ -1,14 +1,24 @@
-apiVersion: v1
-description: "Kubernetes resources for cray-csm-barebones-recipe-install"
-name: "cray-csm-barebones-recipe-install"
-home: "scms/image-recipes"
-# This placeholder string is replaced at build time
-version: @VERSION@
+apiVersion: v2
+name: cray-csm-barebones-recipe-install
+version: 0.0.0
+description: Kubernetes resources for cray-csm-barebones-recipe-install
 keywords:
-  - ims
-  - kiwi
+- ims
```

Comment:
We lost the indentation here and the line below it. Basically, the indentation doesn't seem consistent between keywords, sources, and dependencies.

Reply:
The formatting is consistent. 

`apiVersion: v2
name: cray-csm-barebones-recipe-install
version: 0.0.0
description: Kubernetes resources for cray-csm-barebones-recipe-install
keywords:
- ims
- kiwi
home: https://github.com/Cray-HPE/image-recipes
sources:
- https://github.com/Cray-HPE/image-recipes
dependencies:
- name: cray-import-kiwi-recipe-image
  version: 2.0.0
  repository: https://artifactory.algol60.net/artifactory/csm-helm-charts/
maintainers:
- name: ecozzi-hpe
  email: eric.cozzi@hpe.com
annotations:
  artifacthub.io/images: |
    - name: cray-csm-barebones-recipe-install
      image: artifactory.algol60.net/csm-docker/stable/cray-csm-sles15sp2-barebones-recipe:0.0.0
    - name: cray-product-catalog-update
      image: artifactory.algol60.net/csm-docker/stable/cray-product-catalog-update
  artifacthub.io/license: MIT`